examples/doc-scan/views/pages/success.ejs

```diff
@@ -206,7 +206,29 @@
                     <div id="collapse-watch-list-screening-checks" class="collapse" aria-labelledby="collapse-watch-list-screening-checks">
                         <div class="card-body">
                             <% sessionResult.getWatchlistScreeningChecks().forEach(function(check){ %>
-                            <%- include('partials/check', { check, hasGeneratedProfile: true, hasWatchlistSummary: true }); %>
+                            <%- include('partials/check', { check }); %>
+                            <% }); %>
+                        </div>
+                    </div>
+                </div>
+                <% } %>
+
+                <% if (sessionResult.getWatchlistAdvancedCaChecks().length > 0) { %>
+                <div class="card">
+                    <div class="card-header" id="watch-list-advanced-ca-checks">
+                        <h3 class="mb-0">
+                            <button class="btn btn-link" type="button" data-toggle="collapse"
+                                data-target="#collapse-watch-list-advanced-ca-checks" aria-expanded="true"
+                                aria-controls="collapse-watch-list-advanced-ca-checks">
+                                Watchlist Advanced Checks
+                            </button>
+                        </h3>
+                    </div>
+
+                    <div id="collapse-watch-list-advanced-ca-checks" class="collapse" aria-labelledby="collapse-watch-list-advanced-ca-checks">
+                        <div class="card-body">
+                            <% sessionResult.getWatchlistAdvancedCaChecks().forEach(function(check){ %>
```

Comment:
Not sure I understand why there would be several advanced checks, given you are only requesting one, what am I missing ?

Reply:
That is sort of the pattern followed everywhere.

Reply:
Yep, there might be further checks added in the future, and if it's left like this then we won't need to update the code :slightly_smiling_face: